scripts/devops_tasks/setup_execute_tests.py

```diff
@@ -12,106 +12,176 @@
 import sys
 from pathlib import Path
 import os
+import logging
 
 from common_tasks import process_glob_string, run_check_call
 
-root_dir = os.path.abspath(os.path.join(os.path.abspath(__file__), '..', '..', '..'))
-dev_setup_script_location = os.path.join(root_dir, 'scripts/dev_setup.py')
+logging.getLogger().setLevel(logging.INFO)
 
-# a return code of 5 from pytest == no tests run
-# evaluating whether we want this or not.
-ALLOWED_RETURN_CODES = []
+root_dir = os.path.abspath(os.path.join(os.path.abspath(__file__), "..", "..", ".."))
+dev_setup_script_location = os.path.join(root_dir, "scripts/dev_setup.py")
 
-MANAGEMENT_PACKAGE_IDENTIFIERS = ['mgmt', 'azure-cognitiveservices', 'azure-servicefabric']
+MANAGEMENT_PACKAGE_IDENTIFIERS = [
+    "mgmt",
+    "azure-cognitiveservices",
+    "azure-servicefabric",
+    "azure-nspkg",
+]
 
-def prep_tests(targeted_packages, python_version):
-    print('running test setup for {}'.format(targeted_packages))
-    run_check_call([python_version, dev_setup_script_location, '-p', ','.join([os.path.basename(p) for p in targeted_packages])], root_dir)
 
-def run_tests(targeted_packages, python_version, test_res):
-    # if we are targeting only packages that are management plane, it is a possibility 
+def prep_tests(targeted_packages, python_version):
+    logging.info("running test setup for {}".format(targeted_packages))
+    run_check_call(
+        [
+            python_version,
+            dev_setup_script_location,
+            "-p",
+            ",".join([os.path.basename(p) for p in targeted_packages]),
+        ],
+        root_dir,
+    )
+
+
+def run_tests(targeted_packages, python_version, test_output_location, test_res):
+    err_results = []
+    # if we are targeting only packages that are management plane, it is a possibility
```

Comment:
This comment or one like it might belong closer to line 62

Reply:
Good call. Cleaned.